src/libs/actions/Policy.js

```diff
@@ -1082,6 +1082,8 @@ function createWorkspace(ownerEmail = '', makeMeAdmin = false, policyName = '',
         if (transitionFromOldDot) {
             Navigation.dismissModal(); // Dismiss /transition route for OldDot to NewDot transitions
         }
+        const routeKey = lodashGet(navigationRef.getState(), 'routes[0].state.routes[0].key');
+        Navigation.setParams({reportID: adminsChatReportID}, routeKey);
```

Comment:
Maybe let's add a comment above for  understanding

Reply:
Thanks, are we good here @NikkiWines?

Reply:
Commented above!